build-tools/create-pkg/create-pkg.targets

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\installers\create-installers.targets" />
+  <PropertyGroup>
+    <PkgInstallDir>/</PkgInstallDir>
+    <PkgInstallDir Condition="$([MSBuild]::IsOsPlatform(Linux))">/usr</PkgInstallDir>
+    <PayloadDir>$(OutputPath)/pkgarchive</PayloadDir>
+    <PkgOutputPath>$(OutputPath)/packages</PkgOutputPath>
+  </PropertyGroup>
+  <Target Name="_CopyFilesToPayloadDir"
+      DependsOnTargets="GetXAVersion" >
```

Comment:
There are a lot of `<Copy/>` calls in this target, so if you build this project again with no changes they would always run.

So you could either:
- Use [`<Copy SkipUnchangedFiles="True"/>`](https://docs.microsoft.com/en-us/visualstudio/msbuild/copy-task?view=vs-2017)
- Specify [Inputs and Outputs](https://github.com/xamarin/xamarin-android/blob/master/Documentation/guides/MSBuildBestPractices.md#incremental-builds)

I think either one is fine, and just pick the one that looks cleaner here.

Reply:
@jonathanpeppers if we change the default target back to build do you think the incremental build related changes will be necessary? The `CreatePkg` target only takes about a minute (even on hosted pool machines). I'm a bit wary about not reconstructing the pkg payload folder entirely each time, as for instance, any changes to remove files in the future may not be represented correctly if a local partial paylod already exists.

Reply:
If the default target is `Build`, we definitely want to make sure it isn't copying these files every time.

Looking at the targets in `create-vsix.csproj`, it seems to mostly specify `Inputs` and `Outputs`.

Reply:
"default target is `Build`" != "we copy these files as part of build."

As per 1:1 discussion today, the `Build` target should (1) be the default, but (2) be a no-op; *nothing* should happen as part of the `Build` target.  (Kinda/sorta like almost nothing happens in the `Build` target for `create-vsix.csproj`; it builds a `.dll`, but it does *not* build a `.vsix` file.)

Instead, the only target which actually creates the `.pkg` file, which implies all the file copying, should be the `CreatePkg` target, which can be called explicitly "elsewhere", e.g. [in `build.groovy`](
https://github.com/xamarin/xamarin-android/pull/2733#issuecomment-465302250).